core/base/signedDistanceField/SignedDistanceField.h

```diff
@@ -0,0 +1,720 @@
+/// \ingroup base
+/// \class ttk::SignedDistanceField
+/// \author Julien Tierny <julien.tierny@lip6.fr>
+/// \author Mohamed Amine Kissi <mohamed.kissi@lip6.fr>
+/// \author Mathieu Pont <mathieu.pont@lip6.fr>
+/// \date August 2023
+
+#pragma once
+#include <iostream>
+
+// base code includes
+#include "DataTypes.h"
+#include "Geometry.h"
+#include "ImplicitTriangulation.h"
+#include "SurfaceGeometrySmoother.h"
+#include "VisitedMask.h"
+#include <BoundingVolumeHierarchy.h>
+#include <Debug.h>
+#include <Triangulation.h>
+#include <queue>
+
+namespace ttk {
+
+  class SignedDistanceField : virtual public Debug {
+  public:
+    SignedDistanceField();
+
+    template <typename dataType,
+              typename triangulationType,
+              typename triangulationType2>
+    int execute(dataType *const outputScalars,
+                triangulationType *const triangulation,
+                triangulationType2 *const englobanteTriangulation,
```

Comment:
`englobante` -> `bounding` (thanks for replacing all occurrences)

Reply:
Done in the next commit!